CONTRIBUTING.md

```diff
@@ -77,6 +77,30 @@ When you send a PR, just make sure that:
 that you did not make in your PR, you're doing it wrong.
 * Also don't forget to add a comment when you update a PR with a ping to [the maintainers](https://github.com/orgs/api-platform/people), so he/she will get a notification.
 
+The commit messages must follow the [Conventional Commits specification](https://www.conventionalcommits.org/).
+The following types are allowed:
+
+* `fix`: bug fix
+* `feat`: new feature
+* `docs`: change in the documentation
+* `spec`: spec change
+* `test`: test-related change
+* `perf`: performance optimization
+* `ci`: CI-related change
+* `chore`: updating dependencies and related changes
+
+Examples:
+
+    fix: Fix something
+
+    feat: Introduce X
+
+    feat!: Introduce Y, BC break
```

Comment:
It would be nice to add an example with a scope, like:
```
feat(metadata): add a metadata feature
```

Reply:
We should even make the scope mandatory for Core.